Create Getting started with List draft (#1925)
DevExpress/devextreme-documentation: concepts/05 UI Components/List/00 Getting Started with List/08 Delete Items.md

> To allow users to delete items from the **List**, set the [allowItemDeleting](/Documentation/ApiReference/UI_Components/dxList/Configuration/#allowItemDeleting) property to **true**. Refer to the [Item Deletion](/Documentation/Guide/UI_Components/List/Item_Deletion/) help article for information on deletion modes.

Reviewer: Let's add info about `itemDeleteMode` instead of a link to the article,

> To allow users to delete items from the **List**, set the [allowItemDeleting](/Documentation/ApiReference/UI_Components/dxList/Configuration/#allowItemDeleting) property to **true**. The **List** supports several deletion modes which determine how users delete items. Use the [itemDeleteMode](/Documentation/ApiReference/UI_Components/dxList/Configuration/#itemDeleteMode) property to define the most suitable deletion mode n your case.

Reviewer: ```suggestion
To allow users to delete items from the **List**, set the [allowItemDeleting](/Documentation/ApiReference/UI_Components/dxList/Configuration/#allowItemDeleting) property to **true**. The **List** supports several deletion modes which determine how users delete items. Use the [itemDeleteMode](/Documentation/ApiReference/UI_Components/dxList/Configuration/#itemDeleteMode) property to define the most suitable deletion mode in your case.
```

---
##### jQuery

    <!-- tab: index.js -->
    $(function () {
        $('#list').dxList({
            // ...
            allowItemDeleting: true,
        });

>             // ...

Reviewer: ```suggestion
```

> To allow users to delete items from the **List**, set the [allowItemDeleting](/Documentation/ApiReference/UI_Components/dxList/Configuration/#allowItemDeleting) property to **true**. The **List** supports several deletion modes which determine how users delete items. Use the [itemDeleteMode](/Documentation/ApiReference/UI_Components/dxList/Configuration/#itemDeleteMode) property to define the most suitable deletion mode in your case.

Reviewer 2: The second sentence contains two parts separate by "which" (comma was missing btw), and that's a sign that something might be wrong - either you need to split it into two sentences... Or maybe you should've used "that" instead. In any event "deletion mode" and "how users delete items" are somewhat similar and you can combine them into one expression to avoid the double "delete/deletion" reference. Then you refer to "deletion mode" again in the third sentence. So this all can be optimized to just two sentences instead of three.  

```suggestion
To allow users to delete items from the **List**, set the [allowItemDeleting](/Documentation/ApiReference/UI_Components/dxList/Configuration/#allowItemDeleting) property to **true**. Use the [itemDeleteMode](/Documentation/ApiReference/UI_Components/dxList/Configuration/#itemDeleteMode) property to select the UI elements and/or user actions that remove items.
```

To allow users to delete items from the **List**, set the [allowItemDeleting](/Documentation/ApiReference/UI_Components/dxList/Configuration/#allowItemDeleting) property to **true**. The **List** supports several deletion modes which determine how users delete items. Use the [itemDeleteMode](/Documentation/ApiReference/UI_Components/dxList/Configuration/#itemDeleteMode) property to define the most suitable deletion mode in your case.

---
##### jQuery

    <!-- tab: index.js -->
    $(function () {
        $('#list').dxList({
            // ...
            allowItemDeleting: true,
        });
    });
##### Angular

    <!-- tab: app.component.html -->
    <dx-list...
        [allowItemDeleting]="true">
    </dx-list>

##### Vue

    <!-- tab: App.vue -->
    <template>
        <div id="app-container">
            <DxList...
                :allow-item-deleting="true">
            </DxList>
        </div>
    </template>

    <script>

>     // ...
>     import service from './products.service';
> 
>     export default {
>         // ...
>         data() {
>             // ...
>         },
>     }

Reviewer: ```suggestion
    // ...
```
